### PR TITLE
Cortx 30426 collect journalctl dmesg

### PR DESCRIPTION
**Pull request by DhobaleVishal**
## Problem Statement

- Collect journalctl dmesg.
- Fixed codacy issues from resource utilisation utils. 

## Design

-   For Bug, Describe the fix here.
-   For new test addition post the link for design

## Coding

   Checklist for Author

-   [ ]  Coding conventions are followed and code is formatted (e.g. using pycharm in-built formatter)

## Testing

  Checklist for Author

-   [ ]  New/Affected tests are executed
-   [x]  Attach test execution logs
https://jts.seagate.com/secure/attachment/519328/CORTX-39426_execution_with_journalctl_dmesg.zip

## Review 

  Checklist for Author

-   [x]  JIRA number/GitHub Issue added to PR
-   [x]  PR is self reviewed
-   [x]  Jira and state/status is updated and JIRA is updated with PR link
-   [x]  Check if the description is clear and explained

## Documentation

  Checklist for Author

-   [ ]  Changes done to ReadMe

### CODE REVIEW

File: config/cluster_config.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 "product_family": "LC"
 "product_type": "k8s"
-#K8s based s3 server master details.
+# K8s based s3 server master details.
+# Note: Make sure master/worker credentials are same as we are using them to collect system stat.
 "nodes":
     - "hostname": ""
-      "ip": ""
```
Comment (VikasKumarC) [inline config/cluster_config.yaml:6]: ip information is not needed? we can keep this param for future use as well

File: src/commons/utils/cluster_utils.py [hunk 2]
```diff
@@ -139,28 +140,21 @@ def collect_support_bundles(self, dir_path: str) -> tuple:
...
-    def send_k8s_cmd(
-            self,
-            operation: str,
-            pod: str,
-            namespace: str,
-            command_suffix: str,
-            decode=False,
-            **kwargs) -> bytes:
-        """send/execute command on logical node/pods."""
+    def send_k8s_cmd(self, operation: str, pod: str, namespace: str, command_suffix: str,
```
Comment (VikasKumarC) [inline src/commons/utils/cluster_utils.py:143]: decode option is not needed?

File: src/commons/utils/corio_utils.py [hunk 3]
```diff
@@ -284,19 +283,35 @@ def store_logs_to_nfs_local_server():
         shutil.rmtree(DATA_DIR_PATH)
 
 
-def is_package_installed_local(package_name):
```
Comment (VikasKumarC) [inline src/commons/utils/corio_utils.py:287]: This method is not needed?
Where this logic is shifted?

File: src/commons/utils/nimon_resource_util.py [hunk 4]
```diff
@@ -49,17 +50,17 @@ def monitor_resource_utilisation(action: str):
             host, user, passwd = node["hostname"], node["username"], node["password"]
             server_nodes.extend(host)
             cluster_obj = RemoteHost(host, user, passwd)
-            resp = cluster_obj.execute_command(cm_cmd.K8S_WORKER_NODES)
+            resp = cluster_obj.execute_command(cm_cmd.KUBECTL_GET_WORKERS_NAME)
             LOGGER.debug("response is: %s", str(resp))
             worker_node = resp[1].strip().split("\n")[1:]
             LOGGER.info("worker nodes: %s", str(worker_node))
             server_nodes.extend(worker_node)
     if action == "start":
```
Comment (VikasKumarC) [inline src/commons/utils/nimon_resource_util.py:58]: May we keep these logic in different methods

**Review by VikasKumarC — COMMENTED** [review]
@DhobaleVishal Please have a look at comments and check if we can do better refactor specifically nimon files

File: config/cluster_config.yaml (hunk 1 above)
Comment (DhobaleVishal) [reply config/cluster_config.yaml:6]: We are not using it so removed for now. We can add back whenever needed.

File: src/commons/utils/corio_utils.py (hunk 3 above)
Comment (DhobaleVishal) [reply src/commons/utils/corio_utils.py:287]: This is part of install package method.

File: src/commons/utils/nimon_resource_util.py (hunk 4 above)
Comment (DhobaleVishal) [reply src/commons/utils/nimon_resource_util.py:58]: @akshaym99 will be taking care this in his task.

File: src/commons/utils/cluster_utils.py (hunk 2 above)
Comment (DhobaleVishal) [reply src/commons/utils/cluster_utils.py:143]: This change required to fix codacy, now we are using it from kwargs.

**Review by VikasKumarC — APPROVED** [review]

**Review by kulrk — APPROVED** [review]

**Review by akshaym99 — APPROVED** [review]

**Review by ketanarlulkar — APPROVED** [review]

**Pull request merged**